src/Assert.php

```diff
@@ -817,6 +818,24 @@ public static function uuid($value, $message = '')
         }
     }
 
+    public static function throws(callable $expression, $class, $message = '')
```

Comment:
it has been decided to typehint against `Closure` as it would be the main usage.